README.md

```diff
@@ -1,4 +1,105 @@
-# sacd
+# Service Access Contract Definition (SACD)
+
+## SACD JSON Format
+
+This document provides a basic overview of the SACD JSON format, which is used to define agreements for exchanging services and assets within the DIMO ecosystem. SACDs ensure that involved parties have a mutually agreed-upon understanding of the exchange and its scope, including any compensation. These agreements are designed to be human-readable and can represent various types of agreements, including permissions and payments.
+
+The [`sacd.template.json`](data/sacd.template.json) provides a generic structure for SACD JSON files. Example SACD files for payments ([`sacd.payment.example.json`](data/sacd.payment.example.json)) and permissions ([`sacd.permission.example.json`](sacd.permission.example.json)) demonstrate how this template is used for specific use cases.
+
+### Overview
+
+The SACD JSON follows a consistent structure, as seen in the template:
+
+```json
+{
+  "specversion": "1.0",
+  "timestamp": "0000-00-00T00:00:00Z",
+  "type": "dimo.sacd",
+  "data": {
+    "grantor": {
+      "address": "0x0000000000000000000000000000000000000000",
+      "name": "",
+      "additionalInfo": {}
+    },
+    "grantee": {
+      "address": "0x0000000000000000000000000000000000000000",
+      "name": "",
+      "additionalInfo": {}
+    },
+    "effectiveAt": "0000-00-00T00:00:00Z",
+    "expiresAt": "0000-00-00T00:00:00Z",
+    "additionalDates": {},
+    "agreement": [
+      {
+        "type": "",
+        "asset": "did:::",
+        "": {},
+        "purpose": "",
+        "attachments": [],
+        "signatures": []
+      }
+    ],
+    "extensions": {}
+  }
+}
+```
+
+Key top-level fields include:
+* `specversion`: Specifies the version of the SACD specification being used. Currently, it is "1.0".
+* `timestamp`: The timestamp of when the SACD was created.
+* `type`: Identifies the document as a "dimo.sacd".
+* `data`: Contains the core information about the agreement.
+
+Within the `data` field, the following are typically found:
+* `grantor`: Details about the party granting access or providing the service. This includes their wallet address, an optional human-readable name, and additional information.
+* `grantee`: Details about the party receiving access or the service. Similar to the grantor, this includes their wallet address, an optional name, and additional information.
+* `effectiveAt`: The date and time when the agreement becomes active.
+* `expiresAt`: The date and time when the agreement expires.
```

Comment:
Maybe standard legalese: must be after `effectiveAt`.

Reply:
Like `"notes": "All rights and permissions granted in this agreement are only valid after the effectiveAt date and until the expiresAt date."` in the JSON?

Reply:
No, sorry, I just meant like `effectiveAt < expiresAt`. Ignore this.